recipes/azure-cli/conda_build_config.yaml

```
@@ -0,0 +1,12 @@
+channel_sources:
+  - conda-forge/label/azure-batch_rc,conda-forge
+  - conda-forge/label/azure-mgmt-containerinstance_rc,conda-forge
+  - conda-forge/label/azure-mgmt-datalake-store_dev,conda-forge
+  - conda-forge/label/azure-mgmt-iotcentral_rc,conda-forge
+  - conda-forge/label/azure-mgmt-loganalytics_rc,conda-forge
+  - conda-forge/label/azure-mgmt-rdbms_rc,conda-forge
+  - conda-forge/label/azure-mgmt-servicefabricmanagedclusters_rc,conda-forge
+  - conda-forge/label/azure-mgmt-servicelinker_rc,conda-forge
+  - conda-forge/label/azure-mgmt-signalr_rc,conda-forge
+  - conda-forge/label/azure-mgmt-sqlvirtualmachine_rc,conda-forge
+  - conda-forge/label/azure-mgmt-synapse_rc,conda-forge
```

This should be a single bullet in the form of 
```suggestion
  - conda-forge/label/azure-batch_rc,conda-forge/label/azure-mgmt-containerinstance_rc,…,conda-forge
```